clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -229,7 +229,8 @@ Changes in existing checks
 
 - Improved :doc:`readability-identifier-naming
   <clang-tidy/checks/readability/identifier-naming>` check in `GetConfigPerFile`
-  mode by resolving symbolic links to header files.
+  mode by resolving symbolic links to header files. Fixed handling of Hungarian
```

Comment:
I think it should be separate item in Release notes unrelated to symlinks.

Reply:
One entry per check. One sentence per change.

Reply:
> One entry per check. One sentence per change.

I don't think it is common practice, see https://releases.llvm.org/17.0.1/tools/clang/tools/extra/docs/ReleaseNotes.html I see 6 entries about `readability-identifier-naming`. Does it change recently and documented somewhere?

Reply:
It's not documented, just an "idea". Release notes in Clang-tidy 17 were a little bit unreadable, so in Calng-tidy 18 I push a little bit to unify them: https://releases.llvm.org/18.1.0/tools/clang/tools/extra/docs/ReleaseNotes.html

Reply:
OK, if you think it makes Release Notes more readable.